.eslintrc.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-    'max-len': ['warn', { code: 120 }]
+    'max-len': ['warn', { code: 120, 
+                          ignoreRegExpLiterals: true }]
```

Comment:
I added this for long regular expressions. Thoughts?

Reply:
Yuck... but do we have a choice?

Reply:
Not really! I believe we can do string concatenation and instantiate RegExp(), but that is worse in my opinion!